site/src/docs/about/accessibility/hds-accessibility.mdx

```diff
@@ -15,7 +15,17 @@ import LeadParagraph from '../../../components/LeadParagraph';
 </LeadParagraph>
 
 ## Accessibility in the City of Helsinki digital services
-<p>The basis of HDS accessibility is <ExternalLink href="https://eur-lex.europa.eu/eli/dir/2016/2102/oj">an EU directive on web accessibility</ExternalLink> which mandates that all new public sector websites published by 23.09.2019 have to comply with <ExternalLink href="https://www.w3.org/TR/WCAG20/">WCAG 2.0 Standard at AA Level</ExternalLink>. On 23.09.2020 all websites, new and old, have to meet this criterion. HDS aims to make the life of projects easier by providing components, patterns, and guidelines that meet these requirements.</p>
+
+<p>
+  The basis of HDS accessibility is{' '}
```

Comment:
As Mika commented earlier, it would be nice if we could get rid of the {' '} added by prettier.